### PR TITLE
9.7.0 Release

### PR DESCRIPTION
**Pull request by emilyrohrbough**
Note: The Electron 18 dependency upgrade bumps this release version to 9.7.0. The git branch does not reflect the correct release version.

### CODE REVIEW

**Review by mjhenkes — APPROVED** [review]

**Comment by rockhold** [comment]
The write-up for the changelog looks good. Looking at commits to develop since the last release, though, I just want to double-check that it's intentional that each of the following is currently omitted:
* https://github.com/cypress-io/cypress/pull/21516
* https://github.com/cypress-io/cypress/pull/21547
* https://github.com/cypress-io/cypress/pull/21379
* https://github.com/cypress-io/cypress/pull/21415

**Comment by emilyrohrbough** [comment]
Follow up, https://github.com/cypress-io/cypress/pull/21547 is an update to our tests and does not have a user-facing change to document.

**Review by tbiethman — CHANGES_REQUESTED** [review]
Electron 18 upgrade should be in here, let me see why it's missed

**Comment by emilyrohrbough** [comment]
@tbiethman Looks like it was tagged as 9.7.0

File: content/_changelogs/9.6.2.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## 9.6.2
+
+_Released 5/23/2022_
+
+**Deprecations:**
+
+- The `Cypress.Cookies.preserveOnce()` and `Cypress.Cookies.defaults()` Cypress
+  APIs have been deprecated. In a future release, support for
+  `Cypress.Cookies.preserveOnce()` and `Cypress.Cookies.defaults()` will be
+  removed. Consider using the [`cy.session()`](/api/commands/session) command
```
Comment (jennifer-shehane) [inline content/_changelogs/9.6.2.md:10]: Maybe mention this is experimental.
```suggestion
  removed. Consider using the experimental [`cy.session()`](/api/commands/session) command
```

File: content/api/cypress-api/cookies.md [hunk 2]
```diff
@@ -212,6 +221,7 @@ Cypress.Cookies.defaults({
 
 | Version                                       | Changes                                                                                       |
 | --------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| [9.6.2](/guides/references/changelog#9-6-2)   | Deprecated `preserveOnce` and `defaults`                                                      |
```
Comment (jennifer-shehane) [inline content/api/cypress-api/cookies.md:224]: Make sure to update this release version if we are going with 9.7.0.

File: content/api/cypress-api/cookies.md [hunk 3]
```diff
@@ -2,6 +2,15 @@
 title: Cypress.Cookies
 ---
 
+<Alert type="warning">
+
+⚠️ **`Cypress.Cookies.preserveOnce()` and `Cypress.Cookies.defaults()` are
+deprecated in Cypress 9.6.2**. In a future release, support for
```
Comment (jennifer-shehane) [inline content/api/cypress-api/cookies.md:8]: Update release version.

**Review by jennifer-shehane — APPROVED** [review]

**Pull request merged**